src/ProxyManager/Exception/FileNotWritableException.php

```diff
@@ -30,4 +31,9 @@ public static function fromNotWritableDirectory(string $directory) : self
             $directory
         ));
     }
+
+    public static function exception(FileWriterException $e) : self
+    {
+        return new self($e->getMessage(), (int) $e->getCode(), $e);
```

Comment:
The error code can be safely skipped here: passing it on to the new exception doesn't convey anything useful downstream